session-5/task4.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var (
+		one   = 1
+		two   = 2
+		three = 3
+		four  = 4
+		five  = 5
+		six   = 6
+	)
+	doubleVariadicElements(&one, &two, &three, &four, &five, &six)
```

Comment:
Taskda da main methodda before-after outputun olmasi gosterilmishdi ki, yalniz pointer istifade edek

```suggestion
	fmt.Println("Before doubling:", one, two, three, four, five, six)

	doubleVariadicElements(&one, &two, &three, &four, &five, &six)

	fmt.Println("After doubling:", one, two, three, four, five, six)
```